docs/pages/project/contributing/contributing-rego.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: Contributing to Rego Policies
+permalink: project/contributing/contributing-rego
+abstract: How to contribute to Rego Policies
+language: en
+type: project
+category: contributing
+list: include
+---
+
+## Background
+Meshery has a built-in policy engine, based on [Open Policy Agent (OPA)](https://www.openpolicyagent.org/docs/latest/). Meshery uses the [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/) query language to create these [policies](https://docs.meshery.io/concepts/logical/policies). 
+
+## Prerequisites
+To start contributing to Meshery Policy Engine, make sure you have [OPA CLI](https://www.openpolicyagent.org/docs/latest/#running-opa) installed on your system. You will also need to clone the [Meshery Server](https://github.com/meshery/meshery/) project from Github.
+
+## Components of Meshery Policy Engine
+
+Rego policies are the declarative logic behind the policy engine. These policies define how the underlying engine understands relationships within Meshery Designs. Rego policies can be found [here](https://github.com/meshery/meshery/tree/master/server/meshmodel/kubernetes/v1.25.2/v1.0.0/policies) in the Meshery Github repository.
+
+Currently the Rego policies are invoked from the Go code in Meshery Server. This requires configuring the OPA context as seen [here](https://github.com/meshery/meshkit/blob/master/models/meshmodel/core/policies/rego_policy_relationship.go).
+
+The Meshery Policy Engine is invoked by calling the evaluation endpoint defined [here](https://github.com/meshery/meshery/blob/master/server/handlers/policy_relationship_handler.go). This endpoint is passed a Meshery Design in JSON which it passes to the Meshery Policy Engine for validation.
```

Comment:
Ideally, the URL here is that of one to the Rest API reference for the `evaluate` endpoint.